static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js

```diff
@@ -125,6 +102,16 @@ export const init = (): Promise<void> => {
             }
         );
 
+        onConsentNotification('advertisement', state => {
+            if (state !== null) {
```

Comment:
Prreviously if consent was `null` we'd run `window.googletag.pubads();` is this call unnecessary now?

Reply:
All `window.googletag.pubads();` does is return a reference to the pubads service. It was left in the code before because it was being run alongside a number of other scripts (https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js#L119) as `setPersonalisedAds()` and had to return something. Since we're pushing into the command stack in a different way we no longer need to do that.

Reply:
I thought we were going to map google to some of the AIB purposes? Or do we do that in the lib at 'advertisement' level?

It looks to me that this lib should also pass the whole AIB purposes state to the callback so things like Google  or other can make up their minds independently?

Reply:
@jeteve This piece of work was done before that decision to derive google's consent from the IAB string. After we're done with the UI we'll need to think further on how we want handle the google vs iab part of the integration. For now we'll leave this simple implementation that is just a rework of the previous implementation.